framework/src/modules/network/network.js

```diff
@@ -61,16 +61,16 @@ module.exports = class Network {
 			await this.channel.invoke('app:getApplicationState')
 		);
 
-		const seedPeers = await lookupPeersIPs(this.options.list, true);
-		const blacklistedPeers = this.options.access.blackList
+		const seedPeers = await lookupPeersIPs(this.options.seedPeers, true);
+		const blacklistedPeers = this.options.blacklistedPeers
 			? this.options.access.blackList.map(peer => ({
 					ipAddress: peer.ip,
 					wsPort: peer.wsPort,
 			  }))
 			: [];
 		const p2pConfig = {
 			nodeInfo: initialNodeInfo,
-			hostAddress: this.options.address,
+			hostAddress: this.options.hostAddress,
```

Comment:
```suggestion
			hostAddress: this.options.address,
```

We are using `address` config option for p2p layer from start, no need to change it just for underlying library.

Reply:
This value is not very useful, but I think it's better to keep it consistent?

Reply:
Yes keep it consistent to what it was before, if we see `address` to be changed to `hostAddress` then we will do it for http_api module as well.